isolator/interface.hpp

```diff
@@ -0,0 +1,6 @@
+#ifndef __INTERFACE_HPP__
+#define __INTERFACE_HPP__
+
+#include <isolator/interface.pb.h>
```

Comment:
@djosborne: Could you send me the full build log? Apparently, I can't reproduce the error on my local setup. Also, could you try `../configure --with-mesos=/path/to/mesos/installation --with-protobuf=/path/to/protobuf/installation` and see if that helps?